.taskcat.yml

```diff
@@ -16,18 +16,24 @@ project:
     RemoteAccessCidr: --override--
     S3TAPConfigBucketName: $[taskcat_random-string]-$[taskcat_current_region]
     EKSClusterName: $[taskcat_random-string]
-    NumberOfNodes: 4
-    MaxNumberOfNodes: 4
     TanzuNetUsername: --override--
     TanzuNetPassword: --override--
     TanzuNetApiToken: --override--
     TanzuNetRelocateImages: --override--
     TAPDomainName: --override--
+    TAPClusterArch: --override--
```

Comment:
```suggestion
```

@satya-dillikar Just noticed that this change didn't make it into the last push, which takes precedence over the test parameter values and caused the last run to fail.